src/test/java/redis/clients/jedis/tests/ShardedJedisTest.java

```diff
@@ -20,9 +20,8 @@
   private static HostAndPort redis2 = HostAndPortUtil.getRedisServers().get(1);
 
   /**
-   * Test for "Issue - BinaryShardedJedis.disconnect() may occur memory leak".
```

Comment:
Those formatting diffs could conflict with `removing ShardedJedis PR`, but if you don't mind with it, it's ok